session_0/word_processing.rb

```diff
@@ -18,7 +20,9 @@ def lower_case!(words)
 # words_with_prefix('apple', ['apple', 'ball', 'applesauce']) would
 # return the words 'apple' and 'applesauce'.
 def words_with_prefix(prefix, words)
-  raise NotImplementedError # TODO
+  output = []
+  words.each {|word| if word.start_with?(prefix); output.push(word); end}
```

Comment:
See if select function can be used here: https://www.rubyguides.com/2019/04/ruby-select-method/